corehq/apps/userreports/views.py

```diff
@@ -439,6 +441,28 @@ def preview_data_source(request, domain, config_id):
     return render(request, "userreports/preview_data.html", context)
 
 
+def process_url_params(params, column_descriptions):
+    columns = {column['name']: column for column in column_descriptions}
+    keyword_filters = {}
+    sql_filters = []
+    for key, value in params.items():
+        if key == 'format':
```

Comment:
was thinking a bit more on this - I wonder if we should use namespacing on either the system fields or the user fields to prevent issues like a user wanting to filter on a column called `format`? possibly a character in front of system fields that can't be used to start postgres columns would be an easy-ish solution (maybe `$format`?)